dev-vcs/mercurial/mercurial-5.3.1.ebuild

```diff
@@ -0,0 +1,171 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6  python3_7 )
+PYTHON_REQ_USE="threads(+)"
+
+inherit bash-completion-r1 elisp-common eutils distutils-r1 flag-o-matic
+
+DESCRIPTION="Scalable distributed SCM"
+HOMEPAGE="https://www.mercurial-scm.org/"
+SRC_URI="https://www.mercurial-scm.org/release/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~ppc-aix ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="+chg emacs gpg test tk zsh-completion"
+
+BROKEN_PYTHON="
+	!~dev-python/python-3.5.0
+	!~dev-python/python-3.5.1
+	!~dev-python/python-3.5.2
+	!~dev-python/python-3.6.0
+	!~dev-python/python-3.6.1"
+
+BDEPEND="${BROKEN_PYTHON}"
+
+RDEPEND="${BROKEN_PYTHON}
+	app-misc/ca-certificates
+	dev-python/zstandard[${PYTHON_USEDEP}]
+	gpg? ( app-crypt/gnupg )
+	tk? ( dev-lang/tk )
+	zsh-completion? ( app-shells/zsh )"
+
+DEPEND="emacs? ( >=app-editors/emacs-23.1:* )
+	test? ( app-arch/unzip
+		dev-python/pygments[${PYTHON_USEDEP}] )"
+
+SITEFILE="70${PN}-gentoo.el"
+
+# Too many tests fail #608720
+RESTRICT="test"
+
+python_prepare_all() {
+	# fix up logic that won't work in Gentoo Prefix (also won't outside in
+	# certain cases), bug #362891
+	sed -i -e 's:xcodebuild:nocodebuild:' setup.py || die
+	# Use absolute import for zstd
+	sed -i -e 's/from \.* import zstd/import zstd/' \
```

Comment:
Does this still apply? You could shorten it to `/import zstd/s/from [.]* //` (and yes, please use `[.]` over escaping, as the latter gets confusing when multiple layers of escaping might apply).